docs/src/performance.md

```diff
@@ -42,7 +42,8 @@ For example, the following steps were used to benchmark the changes introduced i
    ```julia
    julia> using BenchmarkTools, Revise; using Trixi
 
-   julia> trixi_include("examples/2d/elixir_euler_sedov_blast_wave.jl")
+   julia> # nowadays "examples/tree_2d_dgsem/elixir_euler_sedov_blast_wave.jl"
+          trixi_include("examples/2d/elixir_euler_sedov_blast_wave.jl")
```

Comment:
I was wondering if we should use the new filename in the `trixi_include`?
But since the rest of the code assumes that the reader checked specific Git SHAs out,
this is more like a blog and not living documentation.

Reply:
Yeah, that was my intention - today, we would also get some different results due to later changes.